### DISCUSSION TITLE
Listening to episode

### DISCUSSION THREAD

**Discussion by Mann202** [gilbarbara/react-spotify-web-playback#163] [General]
Hi, im using your react-spotify-web-playback on my ReactJS Project, i dont have any bug but i wondering is it possible to play an episode with uri like
"spotify:episode:xxxxx"

**Comment by gilbarbara** [comment]
Hey @Mann202 

The last time I tried the Spotify SDK didn't support episode URIs.
Have you tried to set it? They could have changed that.

**Reply by Mann202** [reply]
i tried to set it in your react-spotify-web-playback but found nothing, is your react-spotify-web-playback support it?

**Reply by gilbarbara** [reply]
This library is just a UI wrapper for the Spotify WebPlayback SDK. 
If it doesn't support episodes, there's no way around it.
